### PR TITLE
ECM-431

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-431

### Change description ###

Added new values for cause list reports

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java [hunk 1]
```diff
@@ -144,6 +152,10 @@ public static ListingType getListingTypeFromCaseData(ListingData listingData, Ca
                     ? DocumentHelper.getHearingDuration(hearingType)
                     : " ");
 
+            log.info("checkScotlandReadingDeliberationDay");
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:155]: this log message (and some below as well) can be removed (may be later)

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java [hunk 2]
```diff
@@ -158,6 +170,16 @@ public static ListingType getListingTypeFromCaseData(ListingData listingData, Ca
         }
     }
 
+    private static String checkScotlandReadingDeliberationDay(DateListedType dateListedType, CaseData caseData) {
+        if (caseData.getManagingOffice() == null
+            || isNullOrEmpty(dateListedType.getHearingTypeReadingDeliberation())
+            || dateListedType.getHearingTypeReadingDeliberation().equals("Neither")) {
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:176]: The two Ors can be combined by this: "Neither".equals(dateListedType.getHearingTypeReadingDeliberation())

Comment (jthmcts) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:176]: There's no need for the managingOffice check either really. And why not just call it getScotland....

**Review by jthmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java (hunk 1 above)
Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java]: Removed

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java (hunk 2 above)
Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java]: Refactored

**Review by gauravchawla73463 — APPROVED** [review]

**Pull request merged**